### PR TITLE
fix(rollup-federation-demo): Added installation and start commang for all 3 apps

### PR DESCRIPTION
**Pull request by lexasq**
During my research I didn't manage to befriend `rollup` with yarn workspaces. `rollup-spa` just wasn't able to resolve react and `react-dom` from main folder. That's why this solution doesn't look as good as it could be. 

In general all apps were working, the problem was with misleading README and absence of one command to install and start all of them

### CODE REVIEW

**Comment by ScriptedAlchemy** [comment]
is this ready for review?

**Review by edchai — APPROVED** [review]

**Comment by lexasq** [comment]
> is this ready for review?

Yes, it should be ok by now.

**Review by valorkin — CHANGES_REQUESTED** [review]
use yarn --cwd

**Comment by valorkin** [comment]
name convention ^

File: rollup-federation-demo/package.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "private": true,
+  "scripts": {
+    "install:deps": "yarn install && yarn install:rollup && yarn install:rollup-rs && yarn install:webpack",
```
Comment (valorkin) [inline rollup-federation-demo/package.json]: if you will use workspaces should not be just `yarn install`?

Comment (lexasq) [reply rollup-federation-demo/package.json]: I wrote in the description that rollup can't resolve a dependency from parent folder in workspaces

**Review by valorkin — APPROVED** [review]

**Comment by ScriptedAlchemy** [comment]
fix conflict pm pr

**Pull request closed without merging**